### ISSUE TITLE
`bun install` and `bun add` error `node-hid`

### ISSUE THREAD

**Issue by fdaciuk** [oven-sh/bun#6957]
### What version of Bun is running?

1.0.10+d85bd49d8

### What platform is your computer?

Linux 6.5.10-1-MANJARO x86_64 unknown

### What steps can reproduce the bug?

- Create a project with `bun init`
- Run inside the project directory: `bun add --exact @elgato-stream-deck/node`
- In `index.ts`, put this code:

```ts
import { openStreamDeck } from '@elgato-stream-deck/node'
openStreamDeck()
console.log('test')
```

- Run `bun index.ts`

### What is the expected behavior?

See "test" string in terminal if you have any "Stream Decks" connected.

Otherwhise, see the error: `No Stream Decks are connected`.

### What do you see instead?

This error:

```
➤ bun index.ts
121 |         throw e;
122 |       }
123 |     }
124 |   }
125 |
126 |   err = new Error(
            ^
error: Could not locate the bindings file. Tried:
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/build/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/build/Debug/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/build/Release/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/out/Debug/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/Debug/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/out/Release/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/Release/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/build/default/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/compiled/20.8.0/linux/x64/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/addon-build/release/install-root/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/addon-build/debug/install-root/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/addon-build/default/install-root/HID_hidraw.node
 → /home/daciuk/code/test-streamdeckui/node_modules/node-hid/lib/binding/node-v115-linux-x64/HID_hidraw.node
      at bindings (/home/daciuk/code/test-streamdeckui/node_modules/bindings/bindings.js:126:8)
      at loadBinding (/home/daciuk/code/test-streamdeckui/node_modules/node-hid/nodehid.js:19:16)
      at showdevices (/home/daciuk/code/test-streamdeckui/node_modules/node-hid/nodehid.js:126:4)
      at listStreamDecks (/home/daciuk/code/test-streamdeckui/node_modules/@elgato-stream-deck/node/dist/index.js:16:22)
      at openStreamDeck (/home/daciuk/code/test-streamdeckui/node_modules/@elgato-stream-deck/node/dist/index.js:57:23)
      at /home/daciuk/code/test-streamdeckui/index.ts:2:0
```

### Additional information

If you install dependencies using `yarn` or `npm`, it works as expected.

**Comment by Electroid** [comment]
Duplicate of #4959 (we will fix this)

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway and @paperdave  in #7132

**Comment by fdaciuk** [comment]
There is only one minor issue with the `1.0.18` update: after a fresh install (run `bun i` without `node_modules` and `bun.lockb`) I can see this error message:

![image](https://github.com/oven-sh/bun/assets/487669/251d540d-f15c-4251-aa74-bd042d0166ec)

Looks like a false positive, because the project works fine when I try to run it.

And after another `bun i`, I can see this success message:

![image](https://github.com/oven-sh/bun/assets/487669/650b04af-0d75-4f2d-a1a8-3257643ef295)
